recipes/photospline/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.3" %}
+
+package:
+  name: photospline
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/cnweaver/photospline/archive/v{{ version }}.tar.gz
+  sha256: 7045a631c41489085037b05fac98fd9cad73dc4262b7eead143d09e5f8265dec
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake >=3.1
+    - {{ compiler('c') }}  # [not osx]
```

Comment:
Does this really not need a compiler of macOS?

Reply:
It does require a compiler, but the Travis image ships with one, and I have never been able to get the conda-build compilers to work with CMake on OSX. I can try harder if this is a blocker.

Reply:
Conda-forge only uses it's own compilers now. Try using them and if it fails I can probably help with understanding the error.

Reply:
I had missed the bit in the docs where I had to download an extra SDK myself. Now it works.